build/README.md

```diff
@@ -12,6 +12,20 @@ The following packages must be installed to build releases with lisk-build:
 sudo apt-get install autoconf build-essential git jq libtool moreutils python2.7 tcl8.5 wget zlib1g-dev
 ```
 
+## MacOS
+
+A machine running macOS 10.12 or 10.13 is assumed.
```

Comment:
is required?

Reply:
MacOS is required. We guarantee it works on 10.12 or 10.13; earlier version might or might not work.

Reply:
@MaciejBaj should this be reworded?